airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -4,6 +4,7 @@
 title: AirbyteProtocol
 type: object
 description: AirbyteProtocol structs
+version: 0.2.1
```

Comment:
@gosusnp Since I'm choosing to call the current version of the protocol "v0.2.1"... I figured I should add this to the file somewhere... do you have any thoughts on where it belongs?

Reply:
Good question, I don't think json-schema defines any standard for versions of the schemas.
`version` is probably a good pick for now.

Once we actually duplicate per major, feels like it could be in the `$id`. `https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/models/src/main/resources/airbyte_protocol/v0/airbyte_protocol.yaml`